### PR TITLE
docs(samples): add runReport samples for dimension filters

### PR DESCRIPTION
**Pull request by anweshan**

### CODE REVIEW

**Comment by sofisl** [comment]
I'd like to mere #206 before merging these samples. Thanks!

**Comment by anweshan** [comment]
@sofisl codeowner review is required to merge

File: samples/runReportWithDimensionFilter.js
Comment (sofisl) [inline samples/runReportWithDimensionFilter.js:85]: These print functions look pretty similar. Would it be possible to export it into a single utility function and reuse it throughout the different samples?

Comment (anweshan) [reply samples/runReportWithDimensionFilter.js:85]: The print functions are mostly identical for most samples but the reason they are not reused throughout diferent samples is the guideline that all samples should be self contained (i.e shouldn't call another sample file in the repository with said utility function) as developers tend to copy paste the sample into their own code rather than downloading the entire repository, so each sample has its own print function

**Review by sofisl — APPROVED** [review]

**Pull request merged**
